### PR TITLE
DRUP8-1002 Hide main menu when on a group page

### PR DESCRIPTION
**Pull request by Ruefa**

### CODE REVIEW

File: templates/block/block--superfish--main.html.twig [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+{% if not is_group_page %}
+<div{{ attributes.addClass(classes) }}>
+  {{ title_prefix }}
+  {% if label %}
+    <h2{{ title_attributes }}>{{ label }}</h2>
+  {% endif %}
+  {{ title_suffix }}
+  {% block content %}
+    <div{{ content_attributes.addClass('block__content') }}>
+      {{ is_group_page }}
```
Comment (emerham) [inline templates/block/block--superfish--main.html.twig]: Is this still needed?

File: madrone.theme [hunk 2]
```diff
@@ -306,3 +310,23 @@ function madrone_theme_suggestions_image_style_alter(array &$suggestions, array
 function madrone_theme_suggestions_image_alter(array &$suggestions, array $variables) {
   $suggestions[] = $variables['theme_hook_original'] . '__' . $variables['style_name'];
 }
+
+/**
+ * Helper function to check if current page is associated with a group.
+ */
```
Comment (emerham) [inline madrone.theme:319]: Add documentation for the return type. something like
```/**
 * Helper function to check if current page is associated with a group.
 *
 * @return bool
 *   True if current page is in a group and false if not.
 */
```

**Review by emerham — CHANGES_REQUESTED** [review]
Some minor changes and then we should be good

File: templates/block/block--superfish--main.html.twig (hunk 1 above)
Comment (emerham) [reply templates/block/block--superfish--main.html.twig]: specifically the {{ is_group_page}} on line 43

Comment (Ruefa) [reply templates/block/block--superfish--main.html.twig]: Good catch that was just for debugging

**Review by emerham — APPROVED** [review]
That looks great! thanks for the updates.

**Pull request merged**